src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -20,6 +20,7 @@ public class HelpWindow extends UiPart<Stage> {
     public static final String BASIC_COMMANDS = "1. To add a new student:\n" + "> add n/NAME p/PHONE_NUMBER \n"
             + "2. To find a student in your address book:\n" + "> find NAME\n" + "> find PHONE_NUMBER\n"
             + "3. To view all information of an existing student:\n" + "> view INDEX\n"
+            + "\t NOTE: Editing a student's homework, attendance and session times require student to be in view mode."
```

Comment:
Not a huge thing but "only being able to edit in view mode" actually hasn't been implemented yet, so maybe it's better to update help messages only when a feature is done and working?

Reply:
ok noted,sure